scripts/servers.py

```diff
@@ -193,6 +197,8 @@ def managed_dev_appserver(
         '--dev_appserver_log_level', log_level,
         app_yaml_path
     ]
+    env = {} if env is None else env
```

Comment:
This is probably from this behavior for ``Popen``:

> If env is not None, it must be a mapping that defines the environment variables for the new process; these are used instead of the default behavior of inheriting the current process’ environment. This mapping can be str to str on any platform or bytes to bytes on POSIX platforms much like [os.environ](https://docs.python.org/3/library/os.html#os.environ) or [os.environb](https://docs.python.org/3/library/os.html#os.environb).

per: https://docs.python.org/3/library/subprocess.html#popen-constructor.

Reply:
I also found that the ``json.loads`` line in start.py wouldn't work if no secrets were passed in, so that will need to be fixed, too.

I'll add these fixes to my PR that I'll be sending out to make things a bit easier.